src/components/ValidityCheckScreen/ValidityCheckScreen.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FunctionComponent, useEffect } from "react";
```

Comment:
Just wondering about the folder structure, seemed weird that this is the only one `Screen` component in its own folder? Probably can look into the folder struc in a separate PR.

Reply:
I'll move it into `QrScanner`, we can revisit this again in future